sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mcps.c

```diff
@@ -149,6 +150,12 @@ void gnrc_lorawan_mcps_process_downlink(gnrc_lorawan_t *mac, uint8_t *psdu,
         return;
     }
 
+    /* Check if downlink was received after an uplink that had `ADRACKReq` bit set */
+    if (mac->mcps.adr_ack_cnt >= CONFIG_LORAMAC_DEFAULT_ADR_ACK_LIMIT) {
+        DEBUG("gnrc_lorawan_mcps: ADR_ACK_CNT reset after downlink\n");
+        mac->mcps.adr_ack_cnt = 0;
```

Comment:
just curious, there's no action when this happens?

Reply:
This is already addressed